recipe/build.sh

```diff
@@ -13,22 +13,26 @@ export ESMF_NETCDF="split"
 export ESMF_NETCDF_INCLUDE=${PREFIX}/include
 export ESMF_NETCDF_LIBPATH=${PREFIX}/lib
 
-export ESMF_COMM=mpich3
+export ESMF_CPP="${CC} -E -P -x c"
+export ESMF_F90=${FC}
+export ESMF_CXX=${CXX}
```

Comment:
With these environment variables set, it is no longer necessary to make links to `gcc` and `gfortran`.